Improve LogixNG expression Entry/Exit

danielb987 opened the pull request:
The LogixNG expression `Entry/exit` can now check if a entry/exit is reverse or bidirectional.

![Skärmklipp 2023-11-12 12 23 21](https://github.com/JMRI/JMRI/assets/20255317/029386dc-4d21-42e3-b046-666c7393db77)

dsand47 commented:
**Bidirectional** is a persistent state set by the **Both Way** option in the **Add Entry Exit Points** table.  

**Reverse** is a transient state that can only true when **birectional** and **active** are both true and the sensor icons were selected in reverse sequence.  A better name is **Reversed**, such as **_is reversed_**.

danielb987 commented:
Do you want it this way? Just checking before I commit it. Note the "is is" in the description when viewing the ConditionalNG. But I'm happy with it if you prefer it.

![Skärmklipp 2023-11-12 17 06 51](https://github.com/JMRI/JMRI/assets/20255317/af6c27b2-d67e-417d-a87f-48f8f5df53fb)
![Skärmklipp 2023-11-12 17 07 38](https://github.com/JMRI/JMRI/assets/20255317/f4a301ac-2eb7-429a-8192-8ea9489c0b41)

dsand47 commented:
No, just change "reverse" to "reversed".   For consistency, the name should also be changed in the code.

To me, reverse is an action, reversed is a state.

danielb987 commented:
> To me, reverse is an action, reversed is a state.

Thank you! I agree.

danielb987 commented:
> For consistency, the name should also be changed in the code.

I will do that as well.

danielb987 commented:
> No, just change "reverse" to "reversed". For consistency, the name should also be changed in the code.

Both are done now.

dsand47 commented:
> > No, just change "reverse" to "reversed". For consistency, the name should also be changed in the code.
> 
> Both are done now.

The EntryExit code should also be changed.

danielb987 commented:
> The EntryExit code should also be changed.

It's fixed now.

danielb987 commented:
@dsand47 
Thanks for your approval. I did some small updates after your review so I would be grateful if you could review the last changes.

I renamed the parameter reverse to reversed for the method DestinationPoints.setActiveEntryExit() and I updated the current-draft-note.
